soc/common/riscv-custom/andes/Kconfig

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2025 Andes Technology Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config CPU_HAS_ANDES_EXECIT
+	bool
+	help
+		The AndesCore supports EXEC.IT instruction.
```

Comment:
help text indent is 1x tab followed by 2x spaces

Reply:
Fixed it, thanks.